src/abstractdataframe/show.jl

```diff
@@ -28,47 +28,25 @@ end
 """
     DataFrames.ourshow(io::IO, x::Any, truncstring::Int)
 
-Render a value to an `IO` object compactly and omitting type information, by
-calling 3-argument `show`, or 2-argument `show` if the former contains line breaks.
-Unlike `show`, render strings without surrounding quote marks.
+Render a value to an `IO` object compactly using print.
 `truncstring` indicates the approximate number of text characters width to truncate
 the output (if it is a non-positive value then no truncation is applied).
 """
-function ourshow(io::IO, x::Any, truncstring::Int; styled::Bool=false)
+function ourshow(io::IO, x::T, truncstring::Int; styled::Bool=false) where T
```

Comment:
`T` isn't used right?

Reply:
ah - yes; it was earlier, but we ended up not using it. Fixing.